pkg/master/controller.go

```diff
@@ -263,6 +264,11 @@ func (c *Controller) CreateOrUpdateMasterServiceIfNeeded(serviceName string, ser
 			ClusterIP:       serviceIP.String(),
 			SessionAffinity: api.ServiceAffinityClientIP,
 			Type:            serviceType,
+			SessionAffinityConfig: &api.SessionAffinityConfig{
```

Comment:
Since we have some troubles in setting default value in `pkg/api/v1/defaults.go`, I just can't find a good place to set the default values.